src/NosCore.PathFinder.Gui/PathfinderGui.cs

```diff
@@ -76,7 +77,7 @@ public static void Main()
 
                 var npcMonsters = _npcMonsterDao.LoadAll().ToList();
                 TypeAdapterConfig<MapMonsterDto, GameObject.MapMonster>.NewConfig().ConstructUsing(src => new GameObject.MapMonster(npcMonsters, Logger));
-                TypeAdapterConfig<MapNpcDto, GameObject.MapNpc>.NewConfig().ConstructUsing(src => new GameObject.MapNpc(new Mock<IItemProvider>().Object, new Mock<IGenericDao<ShopDto>>().Object, new Mock<IGenericDao<ShopItemDto>>().Object, npcMonsters, Logger));
+                TypeAdapterConfig<MapNpcDto, GameObject.MapNpc>.NewConfig().ConstructUsing(src => new GameObject.MapNpc(new Mock<IItemProvider>().Object, new Mock<IGenericDao<ShopDto>>().Object, new Mock<IGenericDao<ShopItemDto>>().Object, npcMonsters, Logger, new List<NpcTalkDto>()));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 286 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5306517)